src/pages/workspace/accounting/qbo/import/QuickbooksImportPage.tsx

```diff
@@ -59,7 +59,7 @@ function QuickbooksImportPage({policy}: WithPolicyProps) {
         },
     ];
 
-    if (policy?.connections?.quickbooksOnline.data.country !== CONST.COUNTRY.US) {
+    if (policy?.connections?.quickbooksOnline?.data?.country !== CONST.COUNTRY.US) {
```

Comment:
Yeah - problem here - that we have crash on this line - seem like TS types for data and country is not expecting to be undefined, but from API side it could :-)
just fyi
<img width="1002" alt="Screenshot 2024-05-01 at 11 59 53" src="https://github.com/Expensify/App/assets/28352309/762b6e34-d101-4133-8ba5-a47eca007b56">

Reply:
thanks! Let's update the TS types for data and country to be optional then? to make sure that we're safely accessing them everywhere, not just here.

Reply:
I have updated the type for `data` to be optional. We need to do the same for the `Connection` object: both `quickbooksOnline` and `xero` are optional. Tried to make that change here but caused ts errors on `updatePolicyConnectionConfig`. Let's create a separate issue to handle that